files.go

```diff
@@ -5,124 +5,82 @@ type File struct {
 	Data         []byte
 }
 
-type Files []File
-
-func NewFilesCluster(cluster Cluster) Files {
-	common := newFilesClusterCommon(cluster)
-	master := newFilesClusterMaster(cluster)
-	worker := newFilesClusterWorker(cluster)
-
-	all := Files{}
-	all = append(all, common...)
-	all = append(all, master...)
-	all = append(all, worker...)
-
-	return all
-}
-
-func NewFilesClusterMaster(cluster Cluster) Files {
-	common := newFilesClusterCommon(cluster)
-	master := newFilesClusterMaster(cluster)
-
-	all := Files{}
-	all = append(all, common...)
-	all = append(all, master...)
-
-	return all
-}
-
-func NewFilesClusterWorker(cluster Cluster) Files {
-	common := newFilesClusterCommon(cluster)
-	worker := newFilesClusterWorker(cluster)
-
-	all := Files{}
-	all = append(all, common...)
-	all = append(all, worker...)
-
-	return all
-}
-
-func newFilesClusterCommon(cluster Cluster) Files {
-	return Files{
-		// TODO(r7vme): Only used by Calico and should be removed
```

Comment:
@calvix I dropped this. See https://github.com/giantswarm/certs/blob/master/files.go#L47-L60. Can you check this is fine? How could I figure out? Just checking TC nodes? Masters or workers?

Reply:
I have no idea if this is used anywhere. 

in AWS calico-certs are not needed anymore as we use AWS CNI, similar in Azure but maybe KVM needs it?

Reply:
@giantswarm/team-ludacris how to do about this? Is there an easy way to verify? What about we drop it and at it back in case we have to?

Reply:
It is certainly included in KVM clusters but I can not easily tell if it's necessary:https://github.com/giantswarm/kvm-operator/blob/40e26ab7b3348285b1113b22b3d350fc8142f688/service/controller/cloudconfig/worker_template.go#L70-L85

It is also used in `azure-operator` but again I can't tell if it's needed.

You could merge here and we try to create a cluster with the new import but I am not aware how I can check otherwise.

Reply:
I would say we go with this and when something got missing we bring it back. You can shout at me.